operator/config/manager/manager.yaml

```diff
@@ -39,13 +39,7 @@ spec:
         - command:
             - /manager
           args:
-            - webhook-server
             - --leader-elect
-            # Secure port for the metrics adapter
```

Comment:
these flags are a leftover from when we had the metrics adapter, which now lives in the metrics-operator, in this operator. These are now obsolete and had to be removed as `flags.Parse` was failing due to unknown flags being provided